examples/tictactoe/src/app.tsx

```diff
@@ -16,10 +22,38 @@ export default function TicTacToe() {
 
   return (
     <div className={"outerDiv"}>
-      <Board squares={board} onClick={(i) => handleClick(i)} />
-      <div className="game-info">
-        <div>{status}</div>
-        <HistoryButtons history={history} onClick={jumpTo} />
+      <div className={"headerDiv"}>
+        <img src="../public/logo.png" alt="logo" width="130" height="50" />
+      </div>
+      <div className={"titleDiv"}>
```

Comment:
```suggestion
      <div className="titleDiv">
```
please don't put brackets around strings since they are not needed